tests/Functional/GetRateTest.php

```diff
@@ -17,31 +17,30 @@
 
     public function testGetSpecificRateRecords()
     {
-        /** @var SpecificRateRecord $rates */
-        $rates = FixtureFactory::createPorter()->import(new ImportSpecification(new GetSpecificRate($this->apiKey)))
-            ->findFirstCollection();
+        $rate = FixtureFactory::createPorter()->importOne(new ImportSpecification(new GetSpecificRate($this->apiKey)));
 
-        $rateRecords = $rates->toAssociativeArray();
+        $this->assertCount(1, $rate);
```

Comment:
This is unnecessary because `importOne` guarantees the iterator only contains one result. If it contains more it throws an exception.